src/components/tooltip/index.stories.mdx

```diff
@@ -0,0 +1,17 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import Tooltip from './index.tsx';
+
+<Meta title="Components/Utils/Tooltip" component={Tooltip} />
```

Comment:
I think here should be in the subcategory `Overlay` instead `Utils`. 

```suggestion
<Meta title="Components/Overlay/Tooltip" component={Tooltip} />
```

Please adjust is. Thanks!